app/src/Repository/MsgsRepository.php

```diff
@@ -262,13 +262,13 @@ public function search(?User $user = null, ?int $type = null, ?array $alias = nu
     }
 
     /**
-     * @param ?array{
-     *     sort: string,
-     *     direction: string,
-     * } $sortParams
+     * @param ?array<string, string> $sortParams
```

Comment:
Idem, c’était bien avant

```suggestion
     * @param ?array{
     *     sort: string,
     *     direction: string,
     * } $sortParams
```